docs/whirlpool/docs/03-Whirlpools SDKs/01-Whirlpools/06-Send Transaction.mdx

```diff
@@ -0,0 +1,372 @@
+
+<Tabs groupId="programming-languages">
+  <TabItem value="ts" label="Typescript" default>
+    ```tsx title="sendTransaction.ts"
+    const signedTransaction = await signTransactionMessageWithSigners(transactionMessageWithComputeUnitInstructions)
+    const base64EncodedWireTransaction = getBase64EncodedWireTransaction(signedTransaction);
+
+    let blockHashLifetime = 150n;
+    let currentBlockHeight = latestBlockHash.value.lastValidBlockHeight - blockHashLifetime;
+    while (latestBlockHash.value.lastValidBlockHeight >= currentBlockHeight) {
+      const signature = await rpc.sendTransaction(base64EncodedWireTransaction, {
+        maxRetries: 0n,
+        skipPreflight: true,
+        encoding: 'base64'
+      }).send();
+      const statuses = await rpc.getSignatureStatuses([signature]).send();
+      if (statuses.value[0]) {
+        if (!statuses.value[0].err) {
+          console.log(`Transaction confirmed: ${signature}`);
+          break;
+        } else {
+          console.error(`Transaction failed: ${statuses.value[0].err.toString()}`);
+          break;
+        }
+      }
+      currentBlockHeight = await rpc.getBlockHeight().send();
+      await new Promise(resolve => setTimeout(resolve, 100));
+    }
+    ```
+  </TabItem>
+  <TabItem value="rust" label="Rust">
+    ```rust title="main.rs"
+    #[tokio::main]
+    async fn main() {
+      // ...
+      all_instructions.extend(open_position_instructions.instructions);
+      let message = Message::new(&all_instructions, Some(&wallet.pubkey()));
+
+      let transaction = Transaction::new(&signers, message, recent_blockhash);
+      let transaction_config = RpcSendTransactionConfig {
+        skip_preflight: true,
+        preflight_commitment: Some(CommitmentLevel::Confirmed),
+        max_retries: Some(0),
+        ..Default::default()
+      };
+      let start_time = Instant::now();
+      let timeout = Duration::from_secs(90);
+
+      let send_transaction_result = loop {
+        if start_time.elapsed() >= timeout {
+          break Err(Box::<dyn std::error::Error>::from("Transaction timed out"));
+        }
+        let signature: Signature = rpc
+          .send_transaction_with_config(&transaction, transaction_config)
+          .await
+          .unwrap();
+        let statuses = rpc
+          .get_signature_statuses(&[signature])
+          .await
+          .unwrap()
+          .value;
+        if let Some(status) = statuses[0].clone() {
+          break Ok((status, signature));
+        }
+        sleep(Duration::from_millis(100)).await;
+      };
+      let signature = send_transaction_result.and_then(|(status, signature)| {
+        if let Some(err) = status.err {
+          Err(Box::new(err))
+        } else {
+          Ok(signature)
+        }
+      });
+      println!("Result: {:?}", signature);
+    }
+    ```
+  </TabItem>
+</Tabs>
```

Comment:
Might be nice to add a section about further improvements you could make:
* Sending to multiple rpc nodes concurrently
* Sending to jito block engine and adding a jito tip
* swQoS
* More?